packages/node/src/integrations/contextlines.ts

```diff
@@ -0,0 +1,135 @@
+import { getCurrentHub } from '@sentry/core';
+import { Event, EventProcessor, Integration } from '@sentry/types';
+import { addContextToFrame } from '@sentry/utils';
+import { readFile } from 'fs';
+import { LRUMap } from 'lru_map';
+import { promisify } from 'util';
+
+import { NodeClient } from '../client';
+
+const FILE_CONTENT_CACHE = new LRUMap<string, string | null>(100);
+const DEFAULT_LINES_OF_CONTEXT = 7;
+
+function readTextFileAsync(path: string): Promise<string> {
+  return promisify(readFile)(path, { encoding: 'utf8' });
```

Comment:
I just realized this might be Node 8 only and we support Node 6 lol. We might have to revert :(